### DISCUSSION TITLE
static cache

### DISCUSSION THREAD

**Discussion by wkharrat** [nielsbasjes/yauaa#704] [Q&A] [answered]
is there a downside if i initiate a static cache in servlet (with load on startup) and use it in another servlet . reloading the context is not problem, i can stop and restart tomcat, if i have to do.
With a multiple use of this static cache, is there a memory leak possibility?

**Comment by nielsbasjes** [comment, marked as answer]
This is a library that parses UserAgents, it has nothing to do with Tomcat.
If you have a question about Tomcat then I recommend you go to a forum about that subject.
